### PR TITLE
#1: Updates to the renderer classes

### PR DESCRIPTION
**Pull request by divyarajmasani**
## Summary

- Removes 'show_story_poster' attribute, as we are not using it.
- Updates php tests for the change.
- Removes unnecessary wrapper 'div' from stories renderers.
- Updates single story element classes and styles to add element width.

### CODE REVIEW

File: includes/Stories_Renderer/Renderer.php [hunk 1]
```diff
@@ -350,6 +350,22 @@ protected function get_container_classes() {
 		$container_classes[] = ( ! empty( $this->attributes['align'] ) ) ? sprintf( 'align%1$s', $this->attributes['align'] ) : 'alignnone';
 		$container_classes[] = ( ! empty( $this->attributes['class'] ) ) ? $this->attributes['class'] : '';
 
+		if ( $this->is_view_type( 'grid' ) ) {
+			$container_classes[] = ( ! empty( $this->attributes['number_of_columns'] ) ) ? sprintf( 'columns-%1$s', $this->attributes['number_of_columns'] ) : 'columns-2';
```
Comment (spacedmonkey) [inline includes/Stories_Renderer/Renderer.php:354]:
```suggestion
			$container_classes[] = ( ! empty( $this->attributes['number_of_columns'] ) ) ? sprintf( 'columns-%1$d', $this->attributes['number_of_columns'] ) : 'columns-2';
```

**Review by spacedmonkey — CHANGES_REQUESTED** [review]
There seems to be valid php lint error. Please merge my change and fix lint and I can approve.

**Comment by divyarajmasani** [comment]
@spacedmonkey ,

Resolved phplint errors and accommodated your suggestions, you can take a look again and if looks good.

**Comment by divyarajmasani** [comment]
@spacedmonkey I've updated the PR and fixed the phplint issue as well, if you could get time please check.

**Review by swissspidy — APPROVED** [review]

**Pull request merged**